packages/reactive-element/src/reactive-element.ts

```diff
@@ -1246,6 +1246,12 @@ export abstract class ReactiveElement
   ): void {
     // If we have a property key, perform property update steps.
     if (name !== undefined) {
+      if (DEV_MODE && (name as unknown) instanceof Event) {
+        issueWarning(
+          `request-update-event`,
```

Comment:
We should leave this short code as an empty string unless we intend to actually have a page on lit.dev for it. I do think it's nicer to have a short code if this needs further clarification over the accompanying message. If we intend to keep it, we should make note of it here: https://github.com/lit/lit.dev/issues/1262

Reply:
Ah, good point. Since this is a super specific thing that's fairly self-explanatory when you do hit it, I don't think we need a page on lit.dev for it. Removed the short code.